### ISSUE TITLE
Some HeaderBar Re-organization

### ISSUE THREAD

**Issue by danirabbit** [derekstavis/transmission#24]
We have lots of room in the headerbar! We can bring the design more in-line with the Mac version and expose some basic functions in a better way, I think.
- [ ] Add the New (document-new-symbolic) and Open (document-open-symbolic) items. Might have to double check if GNOME ships "document-new-symbolic". If not, that may have to be provided.
- [ ] Add the view switcher as a pair of .linked buttons with "view-list-symbolic" and "view-list-compact-symbolic". I don't think GNOME ships the list-compact icon. That would also need to be provided.
- [ ] Arrange buttons nicely: [ Add ] [ Open ] [ Remove ] | separator | [Pause ] [ Resume ] | Title Area | [ List ] [ Compact ] | Separator | [Other right side stuff ]

**Comment by derekstavis** [comment]
I wouldn't like to clutter the HeaderBar. But if this stuff is _really_ needed to be in there, we should try to balance the button number (width) as much as we can, so the window title gets always centred.
It's very important to note that Transmission isn't a maximised window program. It should gets as small as the content it displays, that's why OS X implements automatic window resizing.

**Comment by danirabbit** [comment]
I agree that reducing clutter is important. I'm on a 13" display so economic use of space is something that easily effects me :) At the moment, however, I think there's an opposite issue in that there are 3 _huge_ menus with all kinds of stuff in them and that navigation is kind of suffering at the expense of having the headerbar nearly empty. At half my display width, there's still easily 200px of space to either side of the window title. I think that it could be brought more into line with the current Mac OS layout to maintain familiarity across platforms and to aid in some of the menu navigation. Something else to consider is that quite a few headerbar apps are foregoing a window title. It should be immediately apparent what the app is by its layout and contents. So I would personally prioritize clear navigation over showing the title area.

**Comment by derekstavis** [comment]
Yeah, I also like Mac client, the limit popover was inspired by it. :)

Sometimes the title also bugs me, so let's try this menu layout, for this we also remove the title and need to have torrent count somewhere visible by default, as I propose status bar hidden by default.
